src/View/AddOnOptions.py

```diff
@@ -154,7 +154,10 @@ def set_layout(self):
         self.layout.addWidget(self.option_widget, 0, 1, 1, 3)
         try:
             # PySide6 worked on Windows
-            hspacer = QtWidgets.QSpacerItem(QtWidgets.QSizePolicy.Expanding,QtWidgets.QSizePolicy.Minimum)
+            hspacer = QtWidgets.QSpacerItem(1,
```

Comment:
not clear what is an acceptable set of values for the first two arguments (preferred width and height, respectively).

Reply:
I am not in a position to comment at this time.